Examples/Swift/AppDelegate.swift

```diff
@@ -25,8 +25,6 @@
 import UIKit
 import ButtonMerchant
 
-extension String: Error {}
```

Comment:
Unused?

Reply:
Not in the example project -- I added it to the library